### ISSUE TITLE
[Feature Request] Support for VivoBook S14

### ISSUE THREAD

**Issue by aasdelat** [asus-linux-drivers/asus-numberpad-driver#152]
### Short Description

I do not find my Asus  model in the support list, so I make a request for it, if it is possible.

### Additional Info

I was dissapointed after buying the laptop because the numeric numpad was not supported under Linux, but I am very glad someone is taking the job of giving that support. Thanks a lot.

**Comment by aasdelat** [comment]
No, it is not `up5401ea`, but `ux433fa`. The last version of the driver driver (`6.5.0`) correctly detects it.

**Comment by ldrahnik** [comment]
@aasdelat Thank you for the info :)
